### PR TITLE
set addActivity to use firebase

### PR DESCRIPTION
**Pull request by CharlieMantium**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +48,8 @@
     "react-router-dom": "^5.0.1",
     "react-test-renderer": "^16.9.0",
     "redux": "^4.0.4",
+    "redux-mock-store": "^1.5.3",
```
Comment (Lukasz-Semik) [inline package.json]: i think that this package will be used only in tests, so probably it should go to `devDependencies`

File: src/firebase/firebase.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import * as firebase from 'firebase/app';
+import 'firebase/database';
+
+const firebaseConfig = {
```
Comment (Lukasz-Semik) [inline src/firebase/firebase.js]: you could prepare some validating function, which will check if all required keys exists in `process.env`. If not, throw not caught error which will log in terminal lacking key.

File: src/store/activities/actions.js [hunk 3]
```diff
@@ -1,17 +1,28 @@
...
-    id: uuid(),
-    name,
-    room,
-    teacher,
-  },
+  activity,
 });
 
+export const startAddActivity = (activityData = {}) => {
+  return dispatch => {
```
Comment (Lukasz-Semik) [inline src/store/activities/actions.js]: you can omit `return` and `{}`

File: src/store/activities/actions.js [hunk 4]
```diff
@@ -1,17 +1,28 @@
...
+    const { classNo, day, name, room, teacher } = activityData;
+    const activity = { classNo, day, name, room, teacher };
```
Comment (Lukasz-Semik) [inline src/store/activities/actions.js]: i don't understand. `activityData` and `activity` are equal.

File: src/store/activities/actions.js [hunk 5]
```diff
@@ -1,17 +1,28 @@
...
+    return database
+      .ref('activities/items')
+      .push(activity)
+      .then(ref => {
```
Comment (Lukasz-Semik) [inline src/store/activities/actions.js]: you don't have `catch` if error occurres. 

Try to do in `async/awat` fashion, using try-catch block. You can do it in separate PR

File: src/store/activities/actions.js [hunk 6]
```diff
@@ -1,17 +1,28 @@
...
+            id: ref.key,
+            ...activity,
+          }),
+        );
+      });
+  };
+};
+
 export const removeActivity = id => ({
   type: 'REMOVE_ACTIVITY',
```
Comment (Lukasz-Semik) [inline src/store/activities/actions.js]: shouldn't we have constants for `actions types` ?

File: src/tests/setupTests.js [hunk 7]
```diff
@@ -1,5 +1,8 @@
 import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
+import DotEnv from 'dotenv'; // eslint-disable-line import/no-extraneous-dependencies
```
Comment (Lukasz-Semik) [inline src/tests/setupTests.js]: why do we need this eslint ignore?

File: src/store/activities/actions.test.js [hunk 8]
```diff
@@ -1,25 +1,45 @@
-import { addActivity, removeActivity, editActivity } from './actions';
-import { MONDAY } from '../../constants/dates';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import database from '../../firebase/firebase';
```
Comment (Lukasz-Semik) [inline src/store/activities/actions.test.js]: it should be in relative imports group

File: src/firebase/firebase.js (hunk 2 above)
Comment (CharlieMantium) [reply src/firebase/firebase.js]: done, but totally not sure if it's even close to be ok
